### DISCUSSION TITLE
Add support for multiple databases in Laravel models

### DISCUSSION THREAD

**Discussion by rhyudek1** [damms005/devdb-vscode#56] [Ideas]
Within the Laravel models, you can modify the default connection for a specific table

![image](https://github.com/damms005/devdb-vscode/assets/647189/d4a95f81-a849-4642-8b25-01215224ee52)

![image](https://github.com/damms005/devdb-vscode/assets/647189/5a591e90-4195-4580-8e6d-ef9d6553427a)

Is there any chance I can get connection data separately by model?

Regards.

**Comment by damms005** [comment]
This is a nice feature to add!

I will keep this in the logs for when DevDb supports multi-connection/multi-database.

Thank you :hugs:
